### ISSUE TITLE
Rank Embed Buttons

### ISSUE THREAD

**Issue by IQExotic** [Sector-7-Development-Team/Rainbow-Six-Verification#4]
- Add a Button to the [R6 Embed](https://discord.com/channels/678607632692543509/1115374316255715489/1135572721699401901)

1. Add Button
2. Hide Button
3. Show Button

> Right now it's just Update Rank [in Green] and Remove Rank [in Red] both are not Working.
> The Remove Button just removes the role right now. But it should remove the role and the data from the file.
> The Update Rank is obvious ig. Just Update the Rank role.
> Then we need a Hide and Show button that removed the Role from the user same with Show the other way around.
> And Add Button. This should open a Module where the user can put in email and username. And be an alternative for /get-rank.

Tell me if u need help with modules I got a few Links from the Hikari Staff when I reached out to them.

**Comment by Dragos-Lion** [comment]
So on the embed :
Add : Like get-rank with a [modal](https://discordjs.guide/assets/modal-example.1a91ffff.png)
Remove : Remove role and datas from db
Hide and show will be kept in db not to be affect by the update. But what about the lfg ? Like, if they don't want to show their ranks, the lfg command is useless, don't you think so ?

And is the Update button rly necessary ? Since it's done in /lfg + once a day

**Comment by IQExotic** [comment]
No the Lfg is fully usable it just removes the visual role from the user that’s it so when I stake ur Profile I’m not like oh he’s Dia or smth. The Lfg still workes with everything. And the rest yes.

**Comment by Dragos-Lion** [comment]
So for the lfg i remove only the rank, not the stats ?

**Comment by IQExotic** [comment]
No I mean like when I push hide it ![image](https://github.com/Sector-7-Development-Team/Rainbow-Six-Verification/assets/100967619/23394a0c-24c9-4234-a811-50a7681ccf77)it removes my rank roles (see image) and on show it adds it back again. But it doesn’t delete the data the /lfg should not be affected by this. And is there a way to deactivate buttons only for sertant users?

**Comment by Dragos-Lion** [comment]
Nop for the specific thing button. And in /lfg, the rank is shown, so if the member chose hidden, I show it or not ?

**Comment by IQExotic** [comment]
Yes you still show it. The Lfg fetches it from the database right?

**Comment by Dragos-Lion** [comment]
Nop, because it makes the request for the stats so I get the rank and I update it at the same time

**Comment by IQExotic** [comment]
Ye that’s what I mean it gets the id of the user from the database and then fetches the stats.
